### DISCUSSION TITLE
How do I remove the blue box?

### DISCUSSION THREAD

**Discussion by danmartinvela** [zed-industries/zed#12842] [Feature Requests]
Hi, I have noticed the blue box after the last updates, is it possible to remove it?
![Captura desde 2024-06-10 15-17-31](https://github.com/zed-industries/zed/assets/74007579/b572a0d3-984b-4c97-85d2-e7c53cac1095)

**Comment by Moshyfawn** [comment]
There is issue #12723 on this topic.

**Reply by danmartinvela** [reply]
thanks!

**Reply by danmartinvela** [reply]
is this a bug? really ugly!

**Reply by Moshyfawn** [reply]
No, an intended a11y feature. But I've passed on the feedback to possibly disable it via a setting.

There's also an ongoing discussion about this:
- #12725

**Reply by danmartinvela** [reply]
yes, I hope we can disable it on via settings
